cmd/geth/chaincmd.go

```diff
@@ -466,7 +466,7 @@ func dump(ctx *cli.Context) error {
 	stack, _ := makeConfigNode(ctx)
 	defer stack.Close()
 
-	db := utils.MakeChainDatabase(ctx, stack, false)
+	db := utils.MakeChainDatabase(ctx, stack, true)
```

Comment:
- [x] check diff with geth
```suggestion
	db := utils.MakeChainDatabase(ctx, stack, false)
```